18F/identity-idp
# LG-7760: Increase visibility of in-person proofing call-to-action

## 🎫 Ticket

[LG-7760](https://cm-jira.usa.gov/browse/LG-7760)

## 🛠 Summary of changes

Revises the appearance of the in-person proofing call-to-action to appear as an alternative option to "Try again", instead of as a troubleshooting option.

## 📜 Testing Plan

Verify that you see and can opt-in to in-person proofing.

- Navigate to http://localhost:3000
- Sign in
- Navigate to http://localhost:3000/verify
- Complete proofing up to document capture
- Use [attention result test document](https://developers.login.gov/testing/#testing-identity-proofing) for front and back
- Submit

## 👀 Screenshots

_Mobile:_

Before|After
---|---
![localhost_3000_verify_doc_auth_document_capture(iPhone XR) (1)](https://user-images.githubusercontent.com/1779930/197805280-38ca69d2-e046-48d2-9eb1-077684b81f84.png)|![localhost_3000_verify_doc_auth_document_capture(iPhone XR)](https://user-images.githubusercontent.com/1779930/197805286-77f14fa7-606f-4d79-951d-e15e9d195587.png)

_Desktop:_

Before|After
---|---
![localhost_3000_verify_doc_auth_document_capture (1)](https://user-images.githubusercontent.com/1779930/197805274-5435444b-e67f-49b1-8cc2-c0bc66faca78.png)|![localhost_3000_verify_doc_auth_document_capture](https://user-images.githubusercontent.com/1779930/197805277-08260ffd-e228-4cb4-8fac-af9a171f865b.png)
